SPECS/tcl/tcl.spec

```diff
@@ -1,16 +1,14 @@
+%define majorver %(echo %{version} | cut -d. -f1-2)
```

Comment:
@thcrain-msft Linter is looking to put this macro below between `Source0:` and `BuildRequires:`. Seems to be an odd place to put the macro. What are your thoughts?

Reply:
Very odd, haven't seen this behavior before. Feel free to ignore the linter on that. I have some time set aside today to look at linter issues, and I'll add this to my list.